src/main/java/org/monarchinitiative/lirical/svg/Lr2Svg.java

```diff
@@ -248,7 +247,7 @@ private void writeLrBoxes(Writer writer) throws IOException {
                 String color = xstart < midline ? RED : BRIGHT_GREEN;
                 writer.write(String.format("<rect height=\"%d\" width=\"%d\" y=\"%d\" x=\"%d\" " + "stroke-width=\"0\" " +
                         "stroke=\"#000000\" fill=\"%s\" onmouseout=\"hideTooltip();\" " +
-                        "onmouseover=\"showTooltip(evt,\'%s\')\"/>\n", BOX_HEIGHT, (int) boxwidth, currentY, (int) xstart, color, explanationObserved.get(explanationIndex)));
+                        "onmouseover=\"showTooltip(evt,'%s')\"/>\n", BOX_HEIGHT, (int) boxwidth, currentY, (int) xstart, color, explanationObserved.get(explanationIndex)));
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.

Reply:
Thanks, I agree! We will modify.

Reply:
Happy to help.